### PR TITLE
[Cleanup commons-lang] LPD-19395 Remove commons-lang ClassUtils usage

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/IndexerRegistryImpl.java [hunk 1]
```diff
@@ -209,6 +207,36 @@ protected void modified(Map<String, Object> properties) {
 		}
 	}
 
+	private static Class<?>[] _getAllInterfaces(Class<?> clazz) {
```
Comment (tinatian) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/IndexerRegistryImpl.java:210]: @julschong Please use ReflectionUtil.getInterfaces

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
change to use ReflectionUtil.getInterfaces()

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Pull request closed without merging**